package.json

```diff
@@ -163,7 +163,7 @@
     "test:storybook:ci": "concurrently -k -s first -n \"SB,TEST\" -c \"magenta,blue\" \"storybook build --quiet && http-server storybook-static --port 6006 --silent\" \"wait-on 'tcp:127.0.0.1:6006' && test-storybook --coverage\""
   },
   "dependencies": {
-    "@helia/block-brokers": "^3.0.3",
+    "@helia/block-brokers": "^3.0.4",
```

Comment:
addresses https://github.com/ipfs/ipfs-webui/pull/2268#issuecomment-2369384548

even though this is a patch update, I think since we have a package-lock.json, webui isn't getting the fix automatically.